src/mixins/filing-template-mixin.ts

```diff
@@ -30,58 +30,57 @@ export default class FilingTemplateMixin extends Vue {
   @Action setDefineCompanyStepValidity!: ActionBindingIF
   @Action setNameRequestState!: ActionBindingIF
   @Action setOrgPersonList!: ActionBindingIF
-  @Action setFilingId!: ActionBindingIF
   @Action setCertifyState!: ActionBindingIF
   @Action setShareClasses!: ActionBindingIF
   @Action setEffectiveDate!: ActionBindingIF
   @Action setIsFutureEffective!: ActionBindingIF
   @Action setFolioNumber!: ActionBindingIF
   @Action setIncorporationAgreementStepData!: ActionBindingIF
 
-  /** Constructs a filing body, used when saving a filing. */
+  /** Constructs a filing body from store data. Used when saving a filing. */
   buildFiling (): IncorporationFilingIF {
-    // Format DateTime for Filing
-    const effectiveDate = this.stateModel?.incorporationDateTime?.effectiveDate || ''
```

Comment:
We don't need all the fallback values because the initial state properties are always declared (see state-model.ts).

Reply:
cool, so this property is ok to be a passed as `null` in the event that a user selects an immediate filing? The backend/schema has changed a bit throughout development on this property, might have initially been an empty string to pass validators potentially?

Reply:
Not quite.
This method builds a filing object from state data. I'm saying the state data is always declared so we don't need to provide the fallback ( "||" ) values -- which were the same as the initialized values anyways.
If I did this right, the final filing object is the same now as before (except for the new bit).

Reply:
ahh fair enough! In this instance, i believe the declared store value for an `effectiveDate` is null, which conflicts a little with the removed fallback value. So good change! 
If the backend doesn't like null ( i'm can't recall what it expects ), the value could be declared as an empty string in store, either way good clean up!

Reply:
```
    const effectiveDate = this.stateModel.incorporationDateTime.effectiveDate
    const formattedDateTime = effectiveDate &&
      (effectiveDate.toISOString()).replace('Z', '+00:00')
```
effectiveDate is only used in the next line of code (which I didn't change) IF this value is truthy. Both null and '' are falsy, so formattedDateTime isn't assigned (ie, remains undefined). So this is the same functionality as before.